resources/tests.mjs

```diff
@@ -87,6 +87,40 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "TodoMVC-Emoji",
+    url: "resources/todomvc/vanilla-examples/javascript-web-components/dist/index.html",
+    tags: ["todomvc", "experimental"],
```

Comment:
adding experimental tag, since it's a test that uses a workload not in the experimental folder.